src/powfacpy/base_interface.py

```diff
@@ -688,23 +694,20 @@ def get_project_settings(self):
     project_settings_folder = self.get_single_obj("*.SetFold")
     return self.get_single_obj("*.SetPrj", parent_folder=project_settings_folder)
 
-  def _add_selected_variables_for_export(self, results_variables_lists):
+  def _add_selected_variables_for_export(self, results_variables_lists, comres, elmres):
     """Adds selected variables to ComRes for export.
     Arguments:
       results_variables_lists: lists with infos about exported data (results objects,
         elements, variables)
     """
-    elmres = self.app.GetFromStudyCase('ElmRes')
-    comres = self.app.GetFromStudyCase('ComRes')
     comres.iopt_csel = 1 # export only selected variables
-    comres.pResult = None # export only selected variables
     # Insert time
     time_variable_name = powfacpy.PFResultsInterface._get_time_variable_name_from_elmres(elmres)
     
     first_column_is_time = results_variables_lists['variables'][0] == time_variable_name
     if not first_column_is_time: # add time as first column
-      results_variables_lists['result_objects'].insert(0, results_variables_lists['result_objects'][0])
-      results_variables_lists['elements'].insert(0, results_variables_lists['result_objects'][0])
+      results_variables_lists['result_objects'].insert(0, elmres)
+      results_variables_lists['elements'].insert(0, elmres)
```

Comment:
This is to move all items by one index to align with the 'variables' list where the time variable must be added?